### PR TITLE
net class sorting added

### PR DESCRIPTION
**Pull request by svandenb-dev**

### CODE REVIEW

File: pyaedt/edb_core/nets.py [hunk 1]
```diff
@@ -352,6 +352,31 @@ def get_plot_data(
         self._logger.info("Nets Point Generation time %s seconds", round(end_time, 3))
         return objects_lists
 
+    @pyaedt_function_handler()
+    def classify_nets(self, simulation_configuration_object=None):
+        """Sort nets based on SimulationConfiguration object.
+        Nets specified as power/Ground or signal in the Simulation configuration object not initially sorted as
+        this in EDB will be sorted accordingly.
```
Comment (PipKat) [inline pyaedt/edb_core/nets.py:359]:
```suggestion
        If nets specified as ``power/ground`` or ``signal`` in the simulation configuration object are not initially sorted
        in EDB, they are sorted accordingly.
```

File: pyaedt/edb_core/nets.py [hunk 2]
```diff
@@ -352,6 +352,31 @@ def get_plot_data(
...
+
+        Parameters
+        ----------
+        simulation_configuration_object : SimulationConfiguration object
+
+        Returns
+        -------
+        bool
+            True when succeeded, False otherwise.
```
Comment (PipKat) [inline pyaedt/edb_core/nets.py:368]:
```suggestion
            ``True`` when successful, ``False`` when failed.
```

File: pyaedt/edb_core/nets.py [hunk 3]
```diff
@@ -352,6 +352,31 @@ def get_plot_data(
         self._logger.info("Nets Point Generation time %s seconds", round(end_time, 3))
         return objects_lists
 
+    @pyaedt_function_handler()
+    def classify_nets(self, simulation_configuration_object=None):
+        """Sort nets based on SimulationConfiguration object.
```
[the remark this answers is not in the document]
Comment (svandenb-dev) [reply pyaedt/edb_core/nets.py:357]: SimulationConfiguration is the object name

File: pyaedt/edb_core/nets.py [hunk 4]
```diff
@@ -352,6 +352,31 @@ def get_plot_data(
...
+        If nets specified as ``power/ground`` or ``signal`` in the simulation configuration object are not initially sorted
```
Comment (maxcapodi78) [inline pyaedt/edb_core/nets.py:358]:
```suggestion
        If nets specified as ``power/ground`` or ``signal`` in the simulation
        configuration object are not initially sorted.
```

**Review by maxcapodi78 — APPROVED** [review]
LGTM

File: pyaedt/edb_core/nets.py [hunk 5]
```diff
@@ -352,6 +352,31 @@ def get_plot_data(
...
+        in EDB, they are sorted accordingly.
+
+        Parameters
+        ----------
+        simulation_configuration_object : SimulationConfiguration object
```
Comment (maxcapodi78) [inline pyaedt/edb_core/nets.py:363]:
```suggestion
        simulation_configuration_object : :class:`pyaedt.edb_core.EDB_Data.SimulationConfiguration`
```

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**